packages/playground/src/utils/types.ts

```diff
@@ -14,3 +14,39 @@ export interface IPublicConfig {
   gw6?: string;
   domain?: string;
 }
+
+export interface VM {
+  version: number;
+  contractId: number;
+  nodeId: number;
+  name: string;
+  created: number;
+  status: string;
+  message: string;
+  flist: string;
+  publicIP: string;
+  planetary: string;
+  myceliumIP: string;
+  customDomain?: string;
+  interfaces: {
+    network: string;
+    ip: string;
+  }[];
+  capacity: {
+    cpu: number;
+    memory: number;
+  };
+  mounts: any[];
+  env: {
+    SSH_KEY: string;
+  };
+  entrypoint: string;
+  metadata?: any;
+  description?: any;
+  rootfs_size: number;
+  corex: boolean;
+  gpu: any[];
+  deploymentName: string;
+  projectName: string;
+  wireguard: string;
+}
```

Comment:
It's not good if the machine type is not defined in the grid client. If it's not defined, it should be created, and if it is defined, it needs to be exported. However, writing a `GridClient` type in the `Dashboard` is not a good approach.

Reply:
Done.